developing-without-a-build/article.md

```diff
@@ -0,0 +1,266 @@
+# Developing Without a Build
+In the early days of web development, all we needed was a simple file editor and a web server. It was easy for a newcomer to understand the process and get started making their own web pages. Web development has changed a lot since then: the complexity of the tools we use for development has grown just as much as the complexity of the things that we're building on the web.
+
+Nowadays, developers can set up systems which transform their code from one syntax to another, turning it into code that actually runs in the browsers. To aid debugging, source maps need to be configured carefully between the different tools, so that we can still recognize and debug our code in the browser.
+
+Imagine whats it's like for a newcomer or even an experienced developer:
+- They first need to learn a lot of different tools before they can get their code to actually run
+- Their IDE and linter won't understand the syntax of this new framework that was recommended to them by a friend, so they have to look for some extra plugins
+- After that they run into some more errors... Turns out they wanted to keep it simple and not use typescript, but they didn't configure the babel plugins for decorators correctly in the right order so the project didn't compile
+
+It may sound exaggerated, but this experience is common to a lot of developers. You may have jumped through similar hoops yourself.
+
+I feel that's really a shame. One of the key selling points of the web is and always was that's an easy and open format. It should be easy to just get started right away without a lot of configuration, tooling, pomp, and ceremony.
+
+I'm not criticizing any of the tools, they have a role and a purpose. And for a long time it was the only real way to build complex applications on the web, as the web standards and browser implementations were just not there to support modern web development. They really helped push web development forward.
```

Comment:
```suggestion
I'm not criticizing any of those tools, they have all have a role and a purpose. And for a long time it was the only real way to build complex applications on the web, as web standards and browser implementations were just not there to support modern web development. They really helped push web development forward.
```

Reply:
This got outdated, so I can't apply the changes anymore. Applied it manually, thanks!